src/edit/InstanceForm.js

```diff
@@ -181,20 +184,28 @@ class InstanceForm extends React.Component {
   getPaneTitle() {
     const {
       initialValues,
+      stripes,
     } = this.props;
 
     const newInstanceTitle = <FormattedMessage id="ui-inventory.newInstance" />;
 
     const getEditInstanceTitle = () => {
       const publishingInfo = getPublishingInfo(initialValues);
 
+      const isInstanceShared = checkIfUserInCentralTenant(stripes) || initialValues?.source.startsWith(CONSORTIUM_PREFIX);
+      const isInstanceLocal = checkIfUserInMemberTenant(stripes) && ['MARC', 'FOLIO'].includes(initialValues?.source);
+      const instanceType = isInstanceShared ? 'shared' : (isInstanceLocal ? 'local' : '');
+
       return (
         <>
           <AppIcon app="inventory" iconKey="instance" size="small" />
           {' '}
           <FormattedMessage
             id="ui-inventory.editInstance.title"
-            values={{ title: initialValues.title }}
+            values={{
+              title: initialValues.title,
+              instanceType,
```

Comment:
`local`/`shared` should be translated, it's not a constant. You should either translate it before using it as a value for the formatted message or make this selectable in your translations:
`"editInstance.title": "Edit {instanceType, select, shared {shared} local {local} other {}} instance • {title}",`